### ISSUE TITLE
Format parameters in lambda

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#1756]
It looks like ktlint 0.47.1 is not accepting that format in my project:

```
fun combine(a: Any, b: Any, c: (Any, Any) -> Unit) {
  c(a, b)
}

class Foo {
  fun foo() {
    combine(
      Any(),
      Any()
    ) { a,
        b ->
      TODO()
    }
  }
}
```

.editorconfig
```
[*.{kt,kts}]
indent_size = 2
indent_style = space
```

error
```
Foo.kt:11:1: Unexpected indentation (8) (should be 6) (indent)
```

_Originally posted by @eygraber in https://github.com/pinterest/ktlint/issues/1681#issuecomment-1352590673_

**Comment by eygraber** [comment]
After updating to 0.49.0 this is still not working as ktlint expects the formatting to look like this:

```kotlin
fun combine(a: Any, b: Any, c: (Any, Any) -> Unit) {
  c(a, b)
}

class Foo {
  fun foo() {
    combine(
      Any(),
      Any()
    ) { a,
              b -> // Unexpected indentation (8) (should be 14) (standard:indent)
      TODO()
    }
  }
}
```

**Comment by paul-dingemans** [comment]
In 0.49.0 it works correctly for code style ktlint_official. With the other code styles the formatting is indeed incorrect. Will be fixed via #1976 .
